include/irr/asset/IAssetManager.h

```diff
@@ -599,7 +599,7 @@ class IAssetManager : public core::IReferenceCounted
             const char** exts = _writer->getAssociatedFileExtensions();
             for (uint32_t i = 0u; i < IAsset::ET_STANDARD_TYPES_COUNT; ++i)
             {
-                const IAsset::E_TYPE type = IAsset::E_TYPE(1u << i);
+                const IAsset::E_TYPE type = IAsset::E_TYPE((uint64_t)1 << (uint64_t)i);
```

Comment:
actually if you want a 64bit uint constant the suffix is `ull`, e.g. `1ull`

Reply:
I know those literals, right

Reply:
so why the cast?

Reply:
to fix